.changes/591-1095.json

```diff
@@ -17,7 +17,7 @@
     },
     {
         "type": "changed",
-        "description": "updated powerpc64, riscv64, and sparc64 *-linux-gnu images to use a 6.x kernel instead of a 4.x kernel.",
+        "description": "updated powerpc64, riscv64, and sparc64 `*-linux-gnu` images to use a 6.x kernel instead of a 4.x kernel.",
```

Comment:
I quoted this with an inline code block just to avoid any potential formatting issues having an unescaped `*` could have if anything were to change (when validating the resulting changelog). Also, it's consistent with all our other changelogs entries like 727, 745, etc.